pal03377/Athena-New
# Fix config schema detection error on production setups

### Motivation and Context


When running Athena on develop, `module_example` and `module_text_llm` currently fail because of this error:
```
athena-module_example-1             | [INFO] Running in PRODUCTION mode
athena-module_example-1             | Traceback (most recent call last):
athena-module_example-1             |   File "<frozen runpy>", line 198, in _run_module_as_main
athena-module_example-1             |   File "<frozen runpy>", line 88, in _run_code
athena-module_example-1             |   File "/code/module_example/__main__.py", line 94, in <module>
athena-module_example-1             |     app.start()
athena-module_example-1             |   File "/athena/athena/app.py", line 38, in start
athena-module_example-1             |     uvicorn.run(f"{conf.name}.__main__:app", host="0.0.0.0", port=conf.port, proxy_headers=True)
athena-module_example-1             |   File "/code/.venv/lib/python3.11/site-packages/uvicorn/main.py", line 587, in run
athena-module_example-1             |     server.run()
athena-module_example-1             |   File "/code/.venv/lib/python3.11/site-packages/uvicorn/server.py", line 61, in run
athena-module_example-1             |     return asyncio.run(self.serve(sockets=sockets))
athena-module_example-1             |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
athena-module_example-1             |   File "/usr/local/lib/python3.11/asyncio/runners.py", line 190, in run
athena-module_example-1             |     return runner.run(main)
athena-module_example-1             |            ^^^^^^^^^^^^^^^^
athena-module_example-1             |   File "/usr/local/lib/python3.11/asyncio/runners.py", line 118, in run
athena-module_example-1             |     return self._loop.run_until_complete(task)
athena-module_example-1             |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
athena-module_example-1             |   File "/usr/local/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
athena-module_example-1             |     return future.result()
athena-module_example-1             |            ^^^^^^^^^^^^^^^
athena-module_example-1             |   File "/code/.venv/lib/python3.11/site-packages/uvicorn/server.py", line 68, in serve
athena-module_example-1             |     config.load()
athena-module_example-1             |   File "/code/.venv/lib/python3.11/site-packages/uvicorn/config.py", line 467, in load
athena-module_example-1             |     self.loaded_app = import_from_string(self.app)
athena-module_example-1             |                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
athena-module_example-1             |   File "/code/.venv/lib/python3.11/site-packages/uvicorn/importer.py", line 21, in import_from_string
athena-module_example-1             |     module = importlib.import_module(module_str)
athena-module_example-1             |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
athena-module_example-1             |   File "/usr/local/lib/python3.11/importlib/__init__.py", line 126, in import_module
athena-module_example-1             |     return _bootstrap._gcd_import(name[level:], package, level)
athena-module_example-1             |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
athena-module_example-1             |   File "<frozen importlib._bootstrap>", line 1204, in _gcd_import
athena-module_example-1             |   File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
athena-module_example-1             |   File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
athena-module_example-1             |   File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
athena-module_example-1             |   File "<frozen importlib._bootstrap_external>", line 940, in exec_module
athena-module_example-1             |   File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
athena-module_example-1             |   File "/code/module_example/__main__.py", line 13, in <module>
athena-module_example-1             |     @config_schema_provider
athena-module_example-1             |      ^^^^^^^^^^^^^^^^^^^^^^
athena-module_example-1             |   File "/athena/athena/endpoints.py", line 359, in config_schema_provider
athena-module_example-1             |     raise ValueError("@config_schema_provider can only be used once")
athena-module_example-1             | ValueError: @config_schema_provider can only be used once
```

### Description

This PR removes the check as I couldn't find any reliable way of detecting that the decorator is only being used once without also breaking production-enabled setups. I also re-installed all poetry packages because otherwise, there was an issue with the python postgres package not being detected.

### Steps for Testing

- Deploy to the test server
- Make sure that `module_example` and `module_text_llm` are running.
